gridstatus/caiso.py

```diff
@@ -236,16 +241,41 @@ def get_solar_and_wind_forecast_dam(self, date, end=None, verbose=False):
             raw_data=False,
         )
 
-        return self._process_solar_and_wind_forecast_dam(data)
+        return self._process_solar_and_wind_forecast_dam(
+            data,
+            # Day-ahead hourly wind and solar forecast is published at 7:00 AM according
+            # to OASIS.
+            publish_time_offset_from_day_start=pd.Timedelta(hours=7),
+        )
+
+    def _process_solar_and_wind_forecast_dam(
+        self,
+        data,
+        publish_time_offset_from_day_start,
+    ):
+        data = self._add_forecast_publish_time(
```

Comment:
not sure if it makes a difference, but maybe add the publish time after the pivot?

seems more intuitive and performant to me than adding it here, and then including it in the pivot